pom.xml

```
@@ -53,7 +53,7 @@
         <javac.target.version>1.8</javac.target.version>
 
         <version.junit>4.12</version.junit>
-        <version.kotlin>1.3.72</version.kotlin>
+        <version.kotlin>1.4.0</version.kotlin>
```

```suggestion
        <version.kotlin>1.4.10</version.kotlin>
```

kotlin 1.4.10 was released 🎊